pkg/fx/principalresolver/provider.go

```diff
@@ -27,10 +28,14 @@ var Module = fx.Module("principalresolver",
 func NewPrincipalResolver(cfg app.AppConfig) (validator.PrincipalResolver, error) {
 	services := make([]did.DID, 0, 2)
 	if idxSvc := cfg.UCANService.Services.Indexer.Connection; idxSvc != nil {
-		services = append(services, idxSvc.ID().DID())
+		if strings.HasPrefix(idxSvc.ID().DID().String(), "did:web:") {
+			services = append(services, idxSvc.ID().DID())
+		}
 	}
 	if uplSvc := cfg.UCANService.Services.Upload.Connection; uplSvc != nil {
-		services = append(services, uplSvc.ID().DID())
+		if strings.HasPrefix(uplSvc.ID().DID().String(), "did:web:") {
+			services = append(services, uplSvc.ID().DID())
+		}
```

Comment:
only adding web dids to the resolver since they are the only dids that can be resolved, yeah?

Reply:
I did that ^^ and reverted the change.